cmd/tegola_lambda/main.go

```diff
@@ -91,7 +91,12 @@ func init() {
 	// set our server version
 	server.Version = build.Version
 	if conf.Webserver.HostName != "" {
-		server.HostName = string(conf.Webserver.HostName)
+		hostname, err := url.Parse(string(conf.Webserver.HostName))
```

Comment:
@iwpnd you're right it is redundant. I just went back and looked at the code to determine why I did it this way and I think I'm hitting a weird issue that could probably be solved more elegantly:

* The `config` package has a `ValidateAndRegisterParams` routine which calls the `url.Parse()` method on the Hostname and throws an error if the Parse fails. It does not set the value of `Webserver.Hostname`, which is of type `env.String`. 

https://github.com/go-spatial/tegola/blob/e38590a4b2809d5b69b7974c4607d92c4f680c5c/config/config.go#L337-L345

* The `server.Hostname` value is now of type `url.URL` and since `config.Webserver.Hostname` is of type `env.String` we need to parse again. 

I think I opted for the double Parse approach as this code path only gets hit during start up. A more elegant way to solve this would be to add a new type to the `internal/env` package called `URL`. This should be pretty straight forward to do so I will give it a shot. 

Good call out.

Reply:
@iwpnd I added a new commit addressing this comment. LMK what you think.